src/components/Clamp.js

```diff
@@ -125,8 +125,13 @@ export default {
       this.localExpanded = !this.localExpanded
     },
     getLines () {
+      const pureArray = []
+      const clientRects = this.$refs.content.getClientRects()
+      for (let i = 0; i < clientRects.length; i++) {
+        pureArray.push(clientRects[i])
+      }
       return Object.keys(
-        [...this.$refs.content.getClientRects()].reduce(
+        pureArray.reduce(
```

Comment:
We can just use `Array.prototype.slice.call(clientRects)` here without the for loop.

Reply:
Right!